### PR TITLE
Check-in tools used to tag branches, copy branches, and rename branches

### PR DESCRIPTION
**Pull request by addyess**
Created a tool (maybe run via jenkins in the future) to interact with charmed-kubernetes github repos

Due to changes in the sync jobs definitions, post merge, jjb resync will be required

### CODE REVIEW

**Comment by addyess** [comment]
granted, many of these tools replicate the tools available to `jobs/sync-upstream/sync.py`.  There should be some refactor to reuse

**Review by mateoflorido — APPROVED** [review]
PR looks great! LGTM

File: jobs/sync-upstream/sync.py [hunk 1]
```diff
@@ -54,7 +52,7 @@ def cli():
 @click.option(
     "--ancillary-list",
     required=True,
-    help="Path to additionall repos that need to be rebased.",
+    help="Path to additionally repos that need to be rebased.",
```
Comment (stonepreston) [inline jobs/sync-upstream/sync.py:55]:
```suggestion
    help="Path to additional repos that need to be rebased.",
```

Comment (stonepreston) [reply jobs/sync-upstream/sync.py:55]: LGTM aside from the tiny typo here. I like the additional tags for the layers.

**Review by kwmonroe — APPROVED** [review]

**Comment by kwmonroe** [comment]
jjb against `sync-upstream.yaml` has been run

**Pull request merged**